Lib/test/test_asyncio/test_locks.py

```diff
@@ -999,6 +1017,12 @@ def test_release_no_waiters(self):
         sem.release()
         self.assertFalse(sem.locked())
 
+    def test_semaphore_loop_not_running(self):
+        loop = asyncio.new_event_loop()
+        loop.stop()
+        with self.assertWarns(DeprecationWarning):
+            q = asyncio.Semaphore(loop=loop)
+
```

Comment:
Although this technically covers the `BoundedSemaphore` depreciation as well, I think we should explicitly test for it to make sure it's being properly triggered, and so that we can be sure it retains the correct behavior.